kernel/src/process.rs

```diff
@@ -654,7 +703,7 @@ impl<C: Chip> ProcessType for Process<'a, C> {
                         // point the app is no longer valid. The best thing we
                         // can do now is mark the app as still faulted and not
                         // schedule it.
-                        self.state.set(State::Fault);
+                        self.state.set(State::StoppedFaulted);
```

Comment:
This isn't needed, as the call to `terminate()` earlier will have already set the state to `StoppedFaulted`

Reply:
Removed.